docker/common-services.yaml

```diff
@@ -95,11 +95,8 @@ services:
       CAS_DATABASE_NAME: ${CAS_DATABASE_NAME}
       CAS_DEFAULT_USER_ROLE: ${CAS_DEFAULT_USER_ROLE}
       CAS_MONGODB_URI: ${CAS_MONGO_DB_URI:-$FIFTYONE_DATABASE_URI}
-      CAS_URL: ${BASE_URL}
       DEBUG: ${CAS_DEBUG}
-      FIFTYONE_AUTH_MODE: internal
       FIFTYONE_AUTH_SECRET: ${FIFTYONE_AUTH_SECRET}
-      NEXTAUTH_URL: ${BASE_URL}/cas/api/auth
```

Comment:
In `internal` mode we are using `BASE_URL` instead of `AUTH0_BASE_URL` - so `CAS_URL` and `NEXTAUTH_URL` were refactored into the individual `compose.*yaml` files in each directory.

`FIFTYONE_AUTH_MODE` is also unique to the directory, so was refactored into the appropriate `compose.*yaml` files in each directory